### ISSUE TITLE
Hot Module Reload not working after upgrade to webpacker 3.0

### ISSUE THREAD

**Issue by krisrang** [rails/webpacker#724]
Hot Module Reload does not seem to be working correctly after upgrading to webpacker 3.0.
The page is receiving the HMR update and page content is changing but then a full refresh is also done.

**Comment by gauravtiwari** [comment]
@krisrang Oh yes, seems like something we introduced with `watchContentBase` but apparently we seem to have manifest.json in there that will change and it will make the page refresh: 

To fix now: 

```js
// config/webpack/development.js
const environment = require('./environment')
const config = environment.toWebpackConfig()
config.devServer. watchContentBase = false
module.exports = config
```

**Comment by gauravtiwari** [comment]
Released 3.0.1

**Comment by eidge** [comment]
Still seeing this, using webpack 3.0.1 and webpack 3.4.1

Tried @gauravtiwari fix, but no success either.

Any ideas?

**Comment by mattiasottosson** [comment]
Were you able to fix this, @eidge? 

Also tried the above fix with webpacker 3.2.2 and webpack 3.11.0, but full reload still happens

**Comment by coorasse** [comment]
I wrote a detailed explanation on [how to use hot module reloading with Webpacker]( https://medium.com/@coorasse/hot-module-reloading-with-webpacker-b663643a60b1), that could help you.
